modin/_compat/core/py36/pandas_common.py

```diff
@@ -56,18 +58,59 @@ def get_handle(
                 pass
 
 
-def pd_pivot_table(df, **kwargs):  # noqa: PR01, RT01
+def pandas_pivot_table(df, **kwargs):  # noqa: PR01, RT01
     """Perform pandas pivot_table against a dataframe ignoring unsupported args."""
     unsupported_sort = kwargs.pop("sort", None)
     assert unsupported_sort is None
     return df.pivot_table(**kwargs)
 
 
-def pd_convert_dtypes(df, **kwargs):  # noqa: PR01, RT01
+def pandas_convert_dtypes(df, **kwargs):  # noqa: PR01, RT01
     """Perform pandas convert_dtypes against a dataframe or series ignoring unsupported args."""
     unsupported_convert = kwargs.pop("convert_floating", None)
     assert unsupported_convert is None
     return df.convert_dtypes(**kwargs)
 
 
-__all__ = ["get_handle", "pd_pivot_table", "pd_convert_dtypes", "reconstruct_func"]
+def pd_compare(df, **kwargs):
+    result_names = kwargs.pop("result_names", ("self", "other"))
+    assert result_names == (
+        "self",
+        "other",
+    ), "Unsupported argument passed to DataFrame.join: result_names"
```

Comment:
```suggestion
    default_value = ("self", "other")
    result_names = kwargs.pop("result_names", default_value)
    assert (
        result_names == default_value
    ), "Unsupported argument passed to DataFrame.join: result_names"
```

Reply:
@vnlitvinov perhaps this message was lost among many others :)

Reply:
@vnlitvinov, ping

Reply:
fixed